packages/vue/src/components/organisms/SfTabs/_internal/SfTab.vue

```diff
@@ -3,6 +3,7 @@
     <!--@slot Title. Here you should pass a title tab-->
     <slot name="title" v-bind="{ tabClick, isActive, title }">
       <div
+        role="button"
```

Comment:
@peoray if you can add one small improvement that would be great
```suggestion
        role="button"
        :aria-pressed="isActive.toString()"
```